pkg/ddc/alluxio/operations/cached.go

```diff
@@ -61,17 +61,34 @@ func (a AlluxioFileUtils) CachedState() (cached int64, err error) {
 // clean cache with a preset timeout of 60s
 func (a AlluxioFileUtils) CleanCache(path string) (err error) {
 	var (
-		// todo(TrafalgarZZZ): adaptively exec timeout command with proper option according to Alluxio image version
-		command = []string{"timeout", "60", "alluxio", "fs", "free", "-f", path}
-		stdout  string
-		stderr  string
+		releaseVersion   = []string{"cat", "/etc/issue"}
+		commandForUbuntu = []string{"timeout", "60", "alluxio", "fs", "free", "-f", path}
+		commandForAlpine = []string{"timeout", "-t", "60", "alluxio", "fs", "free", "-f", path}
```

Comment:
Could you make `commandForUbuntu` and `commandForAlpine` elegant?

Reply:
> Could you make `commandForUbuntu` and `commandForAlpine` elegant?

Thanks. I have fixed it.